### PR TITLE
Clean logging option

### PR DESCRIPTION
**Pull request by extink**
## Short Description:
Adds an option to enable clean logging. This will strip all meta data from the logging messages like the name of the process, the name of the event, etc.
## Fixes
- #3704

### CODE REVIEW

**Comment by nanogirl21** [comment]
Thank you!

File: configs/path.json.example [hunk 1]
```diff
@@ -1,6 +0,0 @@
-[
-	{"location": "32.087504, 34.806118"},
-	{"location": "Bialik 150, Ramat Gan"},
-	{"location": "Ayalon Highway, Ramat Gan"},
-	{"location": "32.091280, 34.795261"}
-]
```
Comment (k4n30) [inline configs/path.json.example]: Why is this file removed?

**Comment by extink** [comment]
I am not sure why that happened, but I have added that file again ;)

File: configs/path.json.example (hunk 1 above)
Comment (extink) [inline configs/path.json.example]: I really do not know, but I added the file again

**Comment by TheSavior** [comment]
@douglascamata Please review this

**Comment by douglascamata** [comment]
@extink you only updated the coloured logging handler, please update the normal one too. Also, please, make a nested configuration for logs, something like:

```
"log": {
    "color": true,
    "clean_mode": true
}
```

Take a look at the `forts` configuration code to learn how to do it.

**Comment by extink** [comment]
@douglascamata, done!

**Comment by nanogirl21** [comment]
Any idea when this will be merged with the dev? It's pretty low risk as all it's doing is cleaning up the output log.

**Comment by douglascamata** [comment]
I think your merge was messed up. This PR now has 33 changes files with TONS of changes. Was this intended?

**Comment by extink** [comment]
Nope, that was not intended. Cleaned up the commit on this PR. Should be correct now. If not, let me hear!

File: pokemongo_bot/__init__.py [hunk 2]
```diff
@@ -578,32 +579,24 @@ def find_close_cells(self, lat, lng):
...
+        logging.getLogger("requests").setLevel(log_level)
+        logging.getLogger("websocket").setLevel(log_level)
+        logging.getLogger("socketio").setLevel(log_level)
+        logging.getLogger("engineio").setLevel(log_level)
+        logging.getLogger("socketIO-client").setLevel(log_level)
+        logging.getLogger("pgoapi").setLevel(log_level)
+        logging.getLogger("rpc_api").setLevel(log_level)
+
+        if self.config.logging_clean and not self.config.debug:
+            formatter = Formatter(fmt='[%(asctime)s] %(message)s', datefmt='%H:%M:%S')
```
Comment (BriceSD) [inline pokemongo_bot/__init__.py]: @nanogirl21 Do you think it could be possible to have the choice of what to show in the settings ? I like seeing the name, but I totally understand that’s superfluous to most user. Having the choice with this setting by default seems a good compromise to me.

Comment (nanogirl21) [inline pokemongo_bot/__init__.py]: @BriceSD I think you tagged me by mistake, but I would also like a choice if possible.

This is very clustered for me.
`2016-08-13 14:07:53,999 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Tentacool appeared! [CP 308] [Potential 0.6] [A/D/S 10/2/15]`

This is a million times better
`2016-08-13 14:07 [pokemon_appeared] A wild Tentacool appeared! [CP 308] [Potential 0.6] [A/D/S 10/2/15]`

Also, not sure if its the code or my terminal but if the date could be changed to 08-13-2016 (MM/DD/YYYY) and the time to 12 hour format I would be really happy. I know that USA is probably the only ones that use this date/time format.

**Comment by douglascamata** [comment]
Is this PR dead or alive?

**Comment by nanogirl21** [comment]
@extink what is the delay in this? will this EVER be approved for merge?

**Comment by extink** [comment]
@douglascamata: I was waiting for a respone from you guys after my last commit. I had asked if everything was as you guys wished for. If I need to add anything I would like to hear what!

**Comment by BriceSD** [comment]
@extink am I not a guy ? :<

**Comment by extink** [comment]
@BriceSD, did not know you were one of the 'guys'. But if you would like that implemented, I can do that for you ;)

**Comment by k4n30** [comment]
@extink merge conflicts again

**Comment by nanogirl21** [comment]
?

**Comment by nanogirl21** [comment]
@extink  any update??

**Comment by k4n30** [comment]
@nanogirl21 waiting on devs to merge

@douglascamata @TheSavior @BriceSD ping - PR ready for review
Can we ping mention bot?

**Comment by TheSavior** [comment]
Since this one is logging stuff, I defer to @douglascamata.

**Comment by extink** [comment]
@TheSavior & @douglascamata hope you guys can review this one

**Comment by solderzzc** [comment]
close since no  activity in 13days/

**Comment by extink** [comment]
Ehm, why? I have rebased this pr almost every day for the merge...

**Comment by nanogirl21** [comment]
That is batshit insane!

**Comment by k4n30** [comment]
@solderzzc ?

**Comment by solderzzc** [comment]
@k4n30  need manually fix the conflict. hmm. .

**Comment by mjmadsen** [comment]
I think it's that the rebases weren't showing up, so it looked like it was dead.

**Comment by duttonw** [comment]
If there is still issues, I'll try and help make a clean pull request for this. Hopefully with your name on it all later tonight aest time

**Comment by extink** [comment]
And again, rebased the branch ;)

**Comment by mjmadsen** [comment]
I'l wait for CI check then merge. Sorry about the delay @extink

**Comment by mjmadsen** [comment]
CI pasted. Merging!

Thanks for the PR, again sorry it took so long to merge.

**Comment by extink** [comment]
@mjmadsen the rebase wil not usually show up, because it only rebases the current commits, without changing the timestamps. But thanks for the merge!

**Comment by mjmadsen** [comment]
@extink I think that's why he closed it. Looked like it hadn't been touched in quite a few days. Again, sorry for the delay.
